docs/getting-started/GETTING_STARTED.md

```diff
@@ -73,6 +73,17 @@ To complete this tutorial, you need:
     export AIRFLOW__ASTRO_SDK__SQL_SCHEMA=ASTRO_SDK_SCHEMA
     ```
 
+    > **Note:** `AIRFLOW__CORE__ENABLE_XCOM_PICKLING` needs to be enabled for `astro-sdk-python`.
+
+ Currently, custom XCom backends are limited to data types that are json serializable. Since Dataframes are not json serializable, we need to enable XCom pickling to store dataframes.
```

Comment:
@sunank200 @utkarsharma2 @kaxil  Dataframes can be serialised (e.g. to JSON) while keeping the schema by using [to_json](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_json.html) with `orient='table'`:
```
df.to_json(orient=’table’)
```
Pandas also offers the inverse operation, [read_json](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html).

What are your thoughts on using this, allowing users to not have to set `AIRFLOW__CORE__ENABLE_XCOM_PICKLING`?

This would be a bigger change with a ticket / PR, but it may be worth doing it.

Reply:
@tatiana It **can** be serialized manually but not directly by "json" module. The default XCom backend in Airflow uses "json" module, that can't serialize dataframe directly. Custom XCom backend were made available so that users can add that logic based on data types in their custom Xcom backend

Reply:
>What are your thoughts on using this, allowing users to not have to set AIRFLOW__CORE__ENABLE_XCOM_PICKLING?

Can't be done within SDK itself, the built-in json module wouldn't know about Pandas Dataframe or our Table or File class

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/747/commits/df7ea1900bd57197ff137efe6b529ed3ecbbc726)